res/TensorFlowPythonExamples/examples/All/__init__.py

```diff
@@ -0,0 +1,4 @@
+import tensorflow as tf
+
+input_ = tf.compat.v1.placeholder(dtype=tf.bool, shape=(2, 4), name="Hole")
+op_ = tf.raw_ops.All(input=input_, axis=1, keep_dims=False)
```

Comment:
Can we use https://www.tensorflow.org/api_docs/python/tf/reduce_all ? I thought `All` is `ReduceAll`

Reply:
IMHO, it would be better to have both cases, but ``reduce_all`` (which @seanshpark suggested) looks much natural from my view ;-D